Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:6
+
+RUN mkdir -p /usr/src/app
+VOLUME "./:/usr/src/app"
+WORKDIR /usr/src/app
+ADD ./ ./
+RUN npm install 
```

Comment:
For better docker caching using:

```
COPY package.json ./
RUN npm install
COPY ./ ./
```

This only re-runs `npm install` when the `package.json` changed. Currently it would re-run it on every file change.